en/simulation/ignition_gazebo.md

```diff
@@ -18,44 +18,48 @@ See [Simulation](../simulation/README.md) for general information about simulato
 ## Installation (Ubuntu Linux)
 
 :::note
-These instructions were tested on Ubuntu 18.04 and Ubuntu 20.04
+These instructions were tested on Ubuntu 18.04, Ubuntu 20.04, and Ubuntu 22.04
 :::
 
 1. Install the usual [Development Environment on Ubuntu LTS / Debian Linux](../dev_setup/dev_env_linux_ubuntu.md).
-1. Install Ignition Gazebo following the [installation instructions](https://github.com/Auterion/px4-simulation-ignition#readme) (`sudo` may be required):
+1. Install Ignition Gazebo (`sudo` may be required):
+
    ```sh
    sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
    wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
    apt update
-   apt install ignition-edifice
+   apt install ignition-fortress
    ```
 
 ## Running the Simulation
 
 Ignition Gazebo SITL simulation can be conveniently run through a `make` command as shown below:
+
 ```bash
 cd /path/to/PX4-Autopilot
-make px4_sitl ignition
+make px4_sitl gz_x500
```

Comment:
Hmmm, @Jaeyoung-Lim

FMI Why this naming convention? Does this reflect that you'll use `gz_` for ignition frames in future?